src/clj/rems/db/form.clj

```diff
@@ -98,15 +112,20 @@
        (validate-fields)
        (json/generate-string)))
 
+(defn- serialize-formdata [form]
+  (json/generate-string form))
```

Comment:
we usually validate on write as well, cf. `rems.db.blacklist/event->json`

Reply:
True, but actually here the two fields come from the API `CreateFormCommand` which is validated so there is not much to add here except double-checking. Should I do it anyway?

Reply:
I think it's good to have both sides of the validation in place so that we don't forget it when moving the rest of the stuff to `formdata`.

Reply:
Yes, or it could be done there. I'll do it here :+1: